src/storm-pars/transformer/BinaryDtmcTransformer.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "storm/models/sparse/Dtmc.h"
+#include "storm/models/sparse/StandardRewardModel.h"
+
+namespace storm {
+    namespace transformer {
+
+        class BinaryDtmcTransformer {
+
```

Comment:
What are the preconditions for this dtmc transformer?

Reply:
made a docstring :)